### ISSUE TITLE
Summary in elm.json is for other project

### ISSUE THREAD

**Issue by rgrempel** [lukewestby/http-extra#1]
The "summary" entry in your elm.json looks to have been copied from a different project ...

https://github.com/lukewestby/http-extra/blob/e60d5e30ae8c4d5c0734fd5d0c7b5d60fa874a9d/elm.json#L4

**Comment by rgrempel** [comment]
Which makes the entry in the package website look like this, which seems confusing:

![image](https://user-images.githubusercontent.com/64381/45327014-74048d80-b51b-11e8-9e14-0c9c51d6a552.png)

**Comment by jonathanjouty** [comment]
Indeed, I "wasted" a few minutes trying to figure out how this package relates to i18n...
